types/shelljs/index.d.ts

```diff
@@ -14,8 +15,9 @@ import glob = require("glob");
 /**
  * Changes to directory dir for the duration of the script. Changes to home directory if no argument is supplied.
  * @param dir Directory to change in.
+ * @return    Object with shell exit code.
```

Comment:
Might be worth also mentioning `stdout` and `stderr`. (Especially `stderr`, since `stdout` is empty most of the time 🤔)